.scrutinizer.yml

```diff
@@ -0,0 +1,44 @@
+filter:
+    excluded_paths:
+        - tests/
+        - vendor/
+before_commands:
+    - composer install
+tools:
+    php_code_sniffer:
+        config:
+            standard: PSR2
+        extensions:
+            - php
+        enabled: true
+        filter:
+            paths: ["*.php"]
+    php_cpd:
+        min_lines: 5
+        min_tokens: 70
+        names:
+            - '*.php'
+        excluded_dirs:
+            - tests/
+            - vendor/
+        enabled: false
```

Comment:
Since we are probably going to switch to codeclimate, this file should be removed.
@daften ?